ccmlib/scylla_node.py

```diff
@@ -796,3 +797,7 @@ def no_pending_flushes():
     def flush(self):
         self.nodetool("flush")
         self._wait_no_pending_flushes()
+
+    def api(self):
```

Comment:
better to make this property with `@property` decorator and rename it to rest_api.
Usage example: ```node.rest_api.get(...)```
In addition, the instance of the ScyllaREST object will be created on each call to the method, so need to prevent it